src/ui/CozyDialogs/ConfirmDialog.tsx

```diff
@@ -41,34 +71,72 @@ const renderActions = (
   return actions
 }
 
+const ModalContent = ({
+  onClose,
+  title,
+  headerStyle,
+  content,
+  actions
+}: Partial<ConfirmDialogProps>): JSX.Element => (
+  <CozyTheme>
```

Comment:
If we put `<CozyTheme>` here this means that we don't expect to have `inverted` modals in the future. Maybe we should put it in the caller side? (LockScreen)

Reply:
Agreed ✅